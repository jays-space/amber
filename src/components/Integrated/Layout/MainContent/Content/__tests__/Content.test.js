import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import { Content } from "..";

describe("<Content />", () => {
  it("should render correctly", () => {
    render(<Content />);

    const contentComponent = screen.getByRole("main");
    expect(contentComponent).toBeInTheDocument();

    expect(contentComponent).toMatchSnapshot();
  });
});
