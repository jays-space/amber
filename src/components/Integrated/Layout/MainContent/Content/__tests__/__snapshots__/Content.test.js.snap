// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> should render correctly 1`] = `
<main
  class="h-content p-4 md:p-8 md:px-5 lg:px-10 xl:px-14 2xl:px-20"
/>
`;
